index.html

```diff
@@ -0,0 +1,183 @@
+      <a href="https://www.w3.org/TR/wot-thing-description11/#sec-vocabulary-definition">information
+        model</a> and
+      <a href="https://www.w3.org/TR/wot-thing-description11/#table-well-known-operation-types">set
+        of operations</a> [[wot-thing-description11]], as well as being
+      targeted specifically at Web of Things use cases [[wot-usecases]]. It
+      can therefore be thought of as being native to the Web of Things.
+    </p>
+    <p>
+      The sub-protocol defines message payload formats for each of the
+      well-known
+      <a href="https://www.w3.org/TR/wot-thing-description11/#table-well-known-operation-types">
+        operation types</a> defined in the WoT
+      <a href="https://www.w3.org/TR/wot-architecture/#sec-interaction-model">
+        interaction model</a> [[wot-architecture11]], and other messages needed for
+      WebSocket communication.
+    </p>
+    <p>
+      This specification is intended to complement deliverables of the
+      <a href="https://www.w3.org/WoT/wg/">WoT Working Group</a>, including WoT
+      Architecture [[wot-architecture11]], WoT Thing Description
+      [[wot-thing-description11]], WoT Discovery [[wot-discovery]],
+      WoT Binding Templates [[wot-binding-templates]] and WoT Profile
+      [[wot-profile]]. It is intended to implement use cases and requirements
+      defined in the
+      <a
+        href="https://www.w3.org/community/reports/web-thing-protocol/CG-FINAL-web-thing-protocol-requirements-20231101/">Web
+        Thing Protocol Use Cases & Requirements</a> community report.
+    </p>
+    <p>Whilst this document is not on a standards track, the Web Thing Protocol
+      is intended to eventually join a standards track at the W3C or another
+      standards body such as the IETF.
+    </p>
+  </section>
+
+  <section id="conformance">
+  </section>
+
+  <section id="terminology">
+    <h2>Terminology</h2>
+    <p>Fundamental WoT terminology such as Thing or Web Thing, Consumer or WoT
+      Consumer, WoT Thing Description or Thing Description, Interaction Model,
+      Interaction Affordance, Property, Action and Event are defined in
+      the
+      <a href="https://www.w3.org/TR/wot-architecture/#terminology">Terminology</a>
+      section of the WoT Architecture specification [[wot-architecture11]].
+    </p>
+  </section>
+
+  <section id="websocket-connection">
+    <h2>WebSocket Connection</h2>
+    <section id="protocol-handshake">
+      <h3>Protocol Handshake</h3>
+    </section>
+    <section id="protocol-handshake">
+      <h3>WebSocket Re-use</h3>
+    </section>
+  </section>
+
+  <section id="websocket-messages">
+    <h2>WebSocket Messages</h2>
+    <section id="properties">
+      <h3>Properties</h3>
+    </section>
+    <section id="actions">
+      <h3>Actions</h3>
+    </section>
+    <section id="events">
+      <h3>Events</h3>
+    </section>
+    <section id="events">
+      <h3>Other Messages</h3>
```

Comment:
Is this reserved for meta operations or non-WoT messages?

Reply:
Please see the [Strawman Proposal](https://docs.google.com/document/d/1KWv-aQfMgsqBFg0v4rVqzcVvzzisC7y4X4CMUYGc8rE/edit#heading=h.u485la1kuctb). So far the proposed message types for that section include:
- error
- ping
- pong

Reply:
ok thanks. Resolving